_docs/applications/wordpress.md

```diff
@@ -1,6 +1,6 @@
 ---
 layout: guide
-title: StorageOS Docs - WordPress
+title: StorageOS Docs - Swarm with Wordpress
 anchor: applications
 module: applications/wordpress
 # Last reviewed by cheryl.hung@storageos.com on 2017-04-12 - commands work fine but can't connect to WP
```

Comment:
Update this to julian.topley@storageos, and remove the comment about it not working.